### PR TITLE
Fix id keyerror on plugin uninstall

### PR DESCRIPTION
**Pull request by suchmememanyskill**
Please tick as appropriate:
- [ ] I have tested this code on a steam deck or on a PC
    - Completely untested
- [X] My changes generate no new errors/warnings
- [X] This is a bugfix/hotfix
- [ ] This is a new feature

If you're wanting to update a translation or add a new one, please use the weblate page: https://weblate.werwolv.net/projects/decky/

# Description

This fixes issue: An issue has been reported in the steam deck homebrew discord that uninstalling plugins give a keyerror on uninstall. This fixes that

### CODE REVIEW

**Comment by eXhumer** [comment]
semi-colon

**Comment by eXhumer** [comment]
Looks good. I'll test this later today and report back the results.

**Review by eXhumer — APPROVED** [review]
LGTM. No more `KeyError: id` while trying to uninstall/re-install already installed plugins.

File: backend/decky_loader/plugin/sandboxed_plugin.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ async def on_new_message(self, message : str) -> str|None:
 
         if "uninstall" in data:
             self.uninstalling = data.get("uninstall")
+            return None
```
Comment (AAGaming00) [inline backend/decky_loader/plugin/sandboxed_plugin.py:189]: cant you just return

Comment (AAGaming00) [inline backend/decky_loader/plugin/sandboxed_plugin.py:189]: too bad im changing it anyway
```suggestion
            return
```

**Review by AAGaming00 — APPROVED** [review]

**Pull request merged**